cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -115,6 +115,9 @@ type AutoscalingGceClient interface {
 	ResizeMig(GceRef, int64) error
 	DeleteInstances(migRef GceRef, instances []GceRef) error
 	CreateInstances(GceRef, string, int64, []string) error
+
+	// extension point for new operations
```

Comment:
Not sure I understand this comment

Reply:
Yeah, it's very ambiguous to me as well, that basically means that this piece is only exported in order not to reimplement this method in case of interface extension. Do you have any ideas how to replace it?

Reply:
Awesome, the comment seems good from my side, thanks for help! I think that it makes more sense to make it part of the GCE client itself because  semantically it's still a method of a client, even if it has fully utility functionality. 

Also the downsides of this approach that we either need to pass client itself as a parameter (which is relatively same as making it a receiver) or pass http client, poll interval and wait timeout and the parameters we are already passing, which to me seems a bit messy. I personally lean towards leaving it as a receiver function, is that okay from your side?